pkg/model/resource/resource.go

```diff
@@ -148,18 +148,18 @@ func (r *Resource) Update(other Resource) error {
 
 	// Make sure we are not merging resources for different GVKs.
 	if !r.GVK.IsEqualTo(other.GVK) {
-		return fmt.Errorf("unable to update a Resource with another with non-matching GVK")
+		return fmt.Errorf("unable to update a Resource (GVK %+v) with another with non-matching GVK %+v", r.GVK, other.GVK)
 	}
 
 	if r.Plural != other.Plural {
-		return fmt.Errorf("unable to update Resource with another with non-matching Plural")
+		return fmt.Errorf("unable to update Resource (Plural %q) with another with non-matching Plural %q", r.Plural, other.Plural)
```

Comment:
This line is too long
```suggestion
		return fmt.Errorf("unable to update Resource (Plural %q) with another with non-matching Plural %q",
		  r.Plural, other.Plural)
```

Reply:
@estroz committed, PTAL